WordPress/WordPressStatsWidgets/Views/Localization/LocalizableStrings.swift

```diff
@@ -90,7 +90,7 @@ enum LocalizableStrings {
                                                            value: "Create or add a site to see all time stats.",
                                                            comment: "Title of the no site view in all time widget")
 
-    static let noSiteViewThisWeekTitle = AppLocalizedString("widget.thisweek.nosite.view.titlee",
+    static let noSiteViewThisWeekTitle = AppLocalizedString("widget.thisweek.nosite.view.title",
```

Comment:
This is a typo fix, which I noticed by chance. We don't yet have tooling to spot this minor issues 😅